libraries/persistent_store/src/lib.rs

```diff
@@ -12,299 +12,297 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// TODO(ia0): Add links once the code is complete.
+// The documentation is easier to read from a browser:
+// - Run: cargo doc --document-private-items --features=std
+// - Open: target/doc/persistent_store/index.html
+
 //! Store abstraction for flash storage
 //!
 //! # Specification
 //!
-//! The store provides a partial function from keys to values on top of a storage
-//! interface. The store total capacity depends on the size of the storage. Store
-//! updates may be bundled in transactions. Mutable operations are atomic, including
-//! when interrupted.
+//! The [store](Store) provides a partial function from keys to values on top of a
+//! [storage](Storage) interface. The store total [capacity](Store::capacity) depends on the size of
+//! the storage. Store [updates](StoreUpdate) may be bundled in [transactions](Store::transaction).
+//! Mutable operations are atomic, including when interrupted.
 //!
-//! The store is flash-efficient in the sense that it uses the storage lifetime
-//! efficiently. For each page, all words are written at least once between erase
-//! cycles and all erase cycles are used. However, not all written words are user
-//! content: lifetime is also consumed with metadata and compaction.
+//! The store is flash-efficient in the sense that it uses the storage [lifetime](Store::lifetime)
+//! efficiently. For each page, all words are written at least once between erase cycles and all
+//! erase cycles are used. However, not all written words are user content: lifetime is also
```

Comment:
nit: Colon -> Comma, or "lifetime" should have a capital L.

Reply:
Done.